src/admin-views/events-meta-box.php

```diff
@@ -223,7 +223,7 @@ class="tribe-field-timezone tribe-dropdown hide-if-js"
 							id='EventCurrencySymbol'
 							name='EventCurrencySymbol'
 							size='2'
-							value='<?php echo isset( $_EventCurrencySymbol ) ? esc_attr( $_EventCurrencySymbol ) : tribe_get_option( 'defaultCurrencySymbol', '$' ); ?>'
+							value='<?php echo isset( $_EventCurrencySymbol ) && ! empty( $_EventCurrencySymbol ) ? esc_attr( $_EventCurrencySymbol ) : tribe_get_option( 'defaultCurrencySymbol', '$' ); ?>'
```

Comment:
Maybe we can trim the long if statements, and resolve into a flag used, something like `$has_currency_position = isset( $_EventCurrencyPosition ) && ! empty( $_EventCurrencyPosition )`?

Reply:
Since this is super old legacy code that we aim to deprecate completely at some point this year, I would avoid going into any refactor if we can avoid.